templates/features.html

```diff
@@ -98,6 +99,12 @@
   $('#num-features').textContent = e.detail.count;
 });
 
+featureList.addEventListener('unveiled', function() {
+  console.timeStamp && console.timeStamp('features unveiled');
+  console.timeEnd && console.timeEnd('features unveil');
+  unveilStart && ga('send', 'timing', 'features', 'unveil', performance.now() - unveilStart);
```

Comment:
we may want to protect against ios6. IIRC, `window.performance` is defined but `'window.performance.now` isn't :( Believe they shut it off.